Cargo.toml

```diff
@@ -38,14 +38,17 @@ lazy_static = { version = "1.0.2", optional = true }
 thiserror = { version = "1.0.31", optional = true }
 smallvec = { version = "1.0", optional = true }
 
+# https://github.com/open-telemetry/opentelemetry-rust/issues/2165
+tonic = { version = "0.12.3", optional = true }
```

Comment:
I think we should just wait for an upstream fix for this instead of working around it like this.

Reply:
@djc 
As suggested in the [review](https://github.com/tokio-rs/tracing-opentelemetry/pull/169#discussion_r1785396651) and [issue](https://github.com/open-telemetry/opentelemetry-rust/issues/2165#issuecomment-2389963372), after executing `cargo update`, the error was resolved, and the line added to `Cargo.toml`:

```toml
tonic = { version = "0.12.3", optional = true }
```

became unnecessary.

Reply:
That's not actually a fix, that's a workaround. Submitted a fix in https://github.com/open-telemetry/opentelemetry-rust/pull/2168.

Reply:
Thank you for submitting the fix PR. I will update this PR once opentelemetry-proto version 0.26.1 is published.

Reply:
This is released now.